_maps/configs/IndiCaravanTypeA.json

```diff
@@ -0,0 +1,26 @@
+{
+	"map_name": "Caravan-class Modular ship",
+	"map_short_name": "Caravan-class",
+	"map_path": "_maps/shuttles/shiptest/IndiCaravanTypeA.dmm",
```

Comment:
"prefix": "ISV",
"namelists": insert namelists of choice here, recommend GENERAL, SPACE at minimum

Reply:
I knew I was forgetting something.